test/components/util/util.go

```diff
@@ -208,6 +209,7 @@ type PortForwardInfo struct {
 	readyChan     chan struct{}
 	stopChan      chan struct{}
 	portForwarder *portforward.PortForwarder
+	errs          []string
```

Comment:
Maybe `[]error`?

Reply:
The inspiration behind choosing list of string over list of errors is that, I wanted to aggregate the errors and return just one error in the end. Later for aggregating I can just use `strings.Join`, than having to go over the list of errors convert them into string one by one and then joining them using a for loop.